src/test/java/EntityArithmeticFunctionEditExistingRecordTest.java

```diff
@@ -0,0 +1,55 @@
+import base.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utils.ProjectUtils;
+import utils.TestUtils;
+import java.util.List;
+import static utils.TestUtils.jsClick;
+
+public class EntityArithmeticFunctionEditExistingRecordTest extends BaseTest {
+
+    private static final By F1_FIELD = By.id("f1");
+    private static final By F2_FIELD = By.id("f2");
+    private static final By SAVE_BUTTON = By.id("pa-entity-form-save-btn");
+
+    private void createNewRecord() {
+        ProjectUtils.start(getDriver());
+        TestUtils.scrollClick(getDriver(), By.xpath("//p[contains(text(),'Arithmetic Function')]"));
+        getWait().until(ExpectedConditions.elementToBeClickable(By.xpath("//i[contains(text(),'create_new_folder')]"))).click();
+        getWait().until(ExpectedConditions.presenceOfElementLocated(F1_FIELD)).sendKeys("8");
+        findElement(F2_FIELD).sendKeys("9");
+        getWait().until(ExpectedConditions.attributeToBe(By.id("div"), "value", "0.89"));
+        getWait().until(ExpectedConditions.elementToBeClickable(SAVE_BUTTON)).click();
+    }
+
+    private void clickEdit() {
+        jsClick(getDriver(), getDriver().findElement(By.cssSelector(".btn-round")));
+        jsClick(getDriver(), getDriver().findElement(By.xpath("//a[contains(., 'edit')]")));
+    }
+
+    private void editFields() {
+        findElement(F1_FIELD).clear();
+        findElement(F1_FIELD).sendKeys("9");
+        findElement(F2_FIELD).clear();
+        findElement(F2_FIELD).sendKeys("5");
+        getWait().until(ExpectedConditions.attributeToBe(By.id("div"), "value", "1.8"));
+        getDriver().findElement(SAVE_BUTTON).click();
+    }
+
+    @Test
+    public void testEditExistingRecordTest() {
```

Comment:
Fix the name of the test method
```suggestion
    public void testEditExistingRecord() {
```

Reply:
Fixed the name of the test method, thank you